pytorch_toolkit/formula_recognition/configs/medium_config.yml

```diff
@@ -0,0 +1,50 @@
+common:
+  backbone_config:
+    arch: resnext50_32x4d
+    disable_layer_3: true
+    disable_layer_4: true
+  model_path: model_checkpoints/medium_photographed_0185.pth
+  device: cuda
+  vocab_path: vocabs/vocab_medium.json
+train:
+  batch_size: 4
+  learning_rate: 0.000001
+  log_path: logs
+  optimizer: Adam
+  save_dir: model_checkpoints
+  train_paths:
+    - ../../data/formula_recognition/sample_medium_dataset
+  val_path: ../../data/formula_recognition/sample_medium_dataset
+eval:
+  val_path: ../../data/formula_recognition/sample_medium_dataset
+  split_file: validate
+  target_metric: 0.9
```

Comment:
why does it differ from export section?

Reply:
Because in export all images have to have constant input shape. If place the same transformation in the validation set (`TransformCropPad`), the result would be the same in the PyTorch model too.

Reply:
Another workaround is to reload IR model to resize it to the shape of the input image for every image.

Reply:
I would prefer to have the same data pipeline to get pytorch and openvino results as close as possible

Reply:
I've experimented with transformations and got the same result